src/components/Personalization/createApplyPropositions.js

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import composePersonalizationResultingObject from "./utils/composePersonalizationResultingObject";
+import { isNonEmptyArray, isObject } from "../../utils";
+import { DOM_ACTION, HTML_CONTENT_ITEM } from "./constants/schema";
+import PAGE_WIDE_SCOPE from "./constants/scope";
+
+export const EMPTY_PROPOSITIONS = { propositions: [] };
```

Comment:
since this is used only in `validateApplyPropositionsOptions` can we move it there as well?